### PR TITLE
Issue958 results ashrae

### LINKED ISSUE #958: Add reference results from ASHRAE140
**What is the problem?**
The Validation Cases from ASHRAE140-2017  `AixLib.ThermalZones.HighOrder.Validation.ASHRAE140` need to be integrated to CI in order to check the results continuously. 

**How do we want to solve it? Describe the solution you'd like**
The range of accepted results, given by ASHRAE140 for every Case, needs to be included.

### PR DESCRIPTION
**Pull request by konstantinaxanthopoulou**
This closes #958 

- [x] added reference results from ASHRAE140 for AnnualHeatingLoad/CoolingLoad and max/minTemp for free floating cases

### CODE REVIEW

File: AixLib/Utilities/HeatTransfer/HeatToRad.mo [hunk 1]
```diff
@@ -2,6 +2,7 @@ within AixLib.Utilities.HeatTransfer;
 model HeatToRad "Adaptor for approximative longwave radiation exchange with variable surface Area"
   parameter Modelica.SIunits.Emissivity eps = 0.95 "Emissivity";
   parameter Modelica.SIunits.Temperature T0 = Modelica.SIunits.Conversions.from_degC(16) "Initial temperature";
+  Modelica.SIunits.CoefficientOfHeatTransfer hrad "variable radiative heat transfer coefficient";
```
Comment (PMehrfeld) [inline AixLib/Utilities/HeatTransfer/HeatToRad.mo:5]: Why is this variable necessary?

File: AixLib/Utilities/HeatTransfer/HeatToRad.mo [hunk 2]
```diff
@@ -50,6 +51,9 @@ equation
   if not use_A_in then
     A_in_internal =A;
   end if;
+
+  hrad= convPort.Q_flow/(A_in_internal*(convPort.T-radPort.T));
```
Comment (PMehrfeld) [inline AixLib/Utilities/HeatTransfer/HeatToRad.mo:55]: Be careful: Division by zero possible!

**Comment by PMehrfeld** [comment]
This branch `issue958...` is in line with branch `issue883_...`
image

**Comment by konstantinaxanthopoulou** [comment]
There is still one element in the Cases 220, 270, 400 which I could not propagate to the top level. If  this needs to be done, it would be nice if we could talk about it.

**Comment by PMehrfeld** [comment]
@konstantinaxanthopoulou : Sorry for this delay. I need to implement a lot of changes in the window models itself, the window type records, and the solar transmittance / correction models. I am working on it.

**Comment by PMehrfeld** [comment]
I'll merge, even if CI fails. Goes only into branch ìssue883_newTestCases`.

**Pull request merged**